src/code/z_fbdemo_triforce.c

```diff
@@ -10,19 +10,23 @@ void TransitionTriforce_Start(void* thisx) {
         case 2:
             this->transPos = 1.0f;
             return;
+
+        default:
+            this->transPos = 0.03f;
+            return;
```

Comment:
```suggestion
            break;
```
With this brought into the switch-case, these can probably be breaks now

Reply:
It actually doesn't match.

Reply:
Oh, that's surprising! Oh well